.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: ci
+
+on:
+  push:
+    branches: [ 'main', 'stable' ]
+  pull_request:
+    branches: [ 'main', 'stable' ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:        
+  test:
+    runs-on: macos-14
+    strategy:
+      matrix:
+        destination: ['platform=iOS\ Simulator,OS=17.4,name=iPhone\ 15\ Pro']
+        scheme: ['ResearchKit', 'ORKCatalog']
```

Comment:
Lets keep this specific for testing ResearchKit only for now.

Reply:
Do you want me to move all of the `ORKCatalog` test updates out of this PR? I've tested the changes locally and they all pass now

Reply:
Yes, keeping the catalog app as is for now would be best.

Reply:
Reverted